docs/api/data-usage-api.md

```diff
@@ -0,0 +1,58 @@
+# Data Usage API
+This is the API for querying data usage. We currently expose the following actions:
+
+* Get data usage as a total for a company in a month
+* Get data usage for a company in a month, grouped by SIM card collections
+* Get data usage for a company in a month, for a specified SIM card collections
+
+## Endpoint: Get data usage for company in month
+
+**Description**
+
+Get the data usage of a company in a month, as a total value in bytes.
+
+**Endpoint:** `GET /usage/data/{year}/{month}`
+
+**Request Query Parameters**
+
+| Field        | Type    | Description                | Required |
+| ------------ | ------- | -------------------------- | -------- |
+| year         | Integer | Year to see data usage in | Yes       |
+| month        | String  | Month, in text, to see consumption for                  | Yes       |
```

Comment:
Could be nice to align the vertical dividers, and limit the white space of descriptions.